manager/controllers/app/solve_test.go

```diff
@@ -206,15 +206,15 @@ func TestReadWithTransforms(t *testing.T) {
 	account := &v1alpha1.FybrikStorageAccount{}
 	g.Expect(readObjectFromFile("../../testdata/unittests/account-theshire.yaml", account)).NotTo(gomega.HaveOccurred())
 	addCluster(env, multicluster.Cluster{Metadata: multicluster.ClusterMetadata{Region: string(account.Spec.Region)}})
-	p := PathBuilder{Log: &testLog, Env: env, Asset: createReadRequest()}
-	p.Asset.Actions = []taxonomy.Action{{Name: "RedactAction"}}
-	_, err := p.solve()
+	asset := createReadRequest()
+	asset.Actions = []taxonomy.Action{{Name: "RedactAction"}}
+	_, err := solve(env, asset, &testLog)
```

Comment:
I wonder if instead of running tests twice in `make test` we could run different solvers in each test of solve_test.go and compare the Solution objects in addition to the existing checks. We don't really want to re-test the behavior of reconcilers, just the solver.